### PR TITLE
Tls Fastly resources added

### PR DESCRIPTION
**Pull request by lukiwlosek**

### CODE REVIEW

File: Fastly-Tls-Certificate/fastly-tls-certificate.json [hunk 1]
```diff
@@ -0,0 +1,208 @@
+{
+    "typeName": "Fastly::Tls::Certificate",
+    "description": "An example resource schema demonstrating some basic constructs and validation rules.",
```
Comment (joshuadeanhall) [inline Fastly-Tls-Certificate/fastly-tls-certificate.json:3]: Could we update the description

File: Fastly-Tls-Certificate/fastly-tls-certificate.json [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+    "sourceUrl": "https://github.com/aws-cloudformation/aws-cloudformation-rpdk.git",
```
Comment (joshuadeanhall) [inline Fastly-Tls-Certificate/fastly-tls-certificate.json:4]: Can we replace the sourceUrl with the url for this repo?

File: Fastly-Tls-Domain/fastly-tls-domain.json [hunk 3]
```diff
@@ -0,0 +1,138 @@
+{
+    "typeName": "Fastly::Tls::Domain",
+    "description": "An example resource schema demonstrating some basic constructs and validation rules.",
```
Comment (joshuadeanhall) [inline Fastly-Tls-Domain/fastly-tls-domain.json:3]: Could we update the description and sourceURL of this resource also.

File: Fastly-Tls-PrivateKeys/fastly-tls-privatekeys.json [hunk 4]
```diff
@@ -0,0 +1,140 @@
+{
+    "typeName": "Fastly::Tls::PrivateKeys",
+    "description": "An example resource schema demonstrating some basic constructs and validation rules.",
```
Comment (joshuadeanhall) [inline Fastly-Tls-PrivateKeys/fastly-tls-privatekeys.json:3]: Can we update the description and sourceURL for this resource also.

**Review by joshuadeanhall — CHANGES_REQUESTED** [review]

File: Fastly-Tls-PrivateKeys/jest.config.ts [hunk 5]
```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  preset: 'ts-jest',
+  testEnvironment: 'node',
+  transform: {
+      '^.+\\.ts?$': 'ts-jest',
+  },
+  transformIgnorePatterns: ['<rootDir>/node_modules/'],
+};
```
Comment (tbouron) [inline Fastly-Tls-PrivateKeys/jest.config.ts:8]: Same as above

File: Fastly-Tls-PrivateKeys/fastly-tls-privatekeys.json [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+    "handlers": {
+        "create": {
+            "permissions": []
+        },
+        "read": {
+            "permissions": []
+        },
+        "update": {
+            "permissions": []
+        },
```
Comment (tbouron) [inline Fastly-Tls-PrivateKeys/fastly-tls-privatekeys.json:132]: This should be removed as the resource update handle throws a `exceptions.NotUpdatable();`

**Review by tbouron — CHANGES_REQUESTED** [review]
@lukiwlosek Looks good! There are just few snags I found, see below

File: Fastly-Tls-Certificate/inputs/inputs_1_update.json [hunk 7]
```diff
@@ -0,0 +1,7 @@
...
+        "Name": "Tls name"
```
[the remark this answers is not in the document]
Comment (lukiwlosek) [reply Fastly-Tls-Certificate/inputs/inputs_1_update.json:5]: Cannot test as Attributes is an object, and there is an issue with ctv where we cannot update nested objects unfortunately

**Review by joshuadeanhall — APPROVED** [review]

**Pull request merged**